node-bitvavo-api.js

```diff
@@ -549,9 +553,10 @@ let api = function Bitvavo () {
       return privateRequest('/order', '', callback, body, 'POST')
     },
 
-    getOrder: function (symbol = '', orderId = '', callback = false) {
-      let params = { 'market': symbol, 'orderId': orderId }
-      let postfix = createPostfix(params)
+    getOrder: function (market = '', options = {}, callback = false) {
+      if (!isObject(options)) throw new Error('Second parameter "options" in "getOrder" should be a valid object');
```

Comment:
This is breaking backwards capability. Let's please do something like this:
```js
if (typeof options === 'string') {
  options = { orderId: options };
}
```
So that when upgrading, existing code won't break.